src/App.vue

```diff
@@ -210,6 +213,35 @@ export default {
         console.warning("fail to open service");
       }
     },
+    filterTag: function (filter) {
+      this.showMenu = true;
+      this.$nextTick(() => {
+        document.getElementById("searchBox").focus();
+      });
+      this.filter = filter;
+
+      if (!filter) {
+        this.services = this.config.services;
+        return;
+      }
+
+      const searchResultItems = [];
+      for (const group of this.config.services) {
+        for (const item of group.items) {
+          if (this.matchesFilter(item)) {
+            searchResultItems.push(item);
+          }
+        }
+      }
+
+      this.services = [
+        {
+          name: filter,
+          icon: "fas fa-search",
+          items: searchResultItems,
+        },
+      ];
```

Comment:
Can you call the existing filter logic?
```suggestion
      this.filterServices();
```

Reply:
Let me check and see if it still works.